webcompat/static/js/lib/issues.js

```diff
@@ -371,7 +371,8 @@ issues.MainView = Backbone.View.extend({
     $(document.body).addClass('language-html');
     var issueNum = {number: issueNumber};
     this.issue = new issues.Issue(issueNum);
-    this.comments = new issues.CommentsCollection([]);
+    this.commentPage = {page: 1};
+    this.comments = new issues.CommentsCollection([this.commentPage]);
```

Comment:
I'm confused why we're passing in an array here, and then over in comments.js, we're just grabbing the 0th index of that. So why do we need an array at all?

Also, `this.commentPage` isn't used anywhere else, so why not just pass in the object directly to the collections, i.e., 

`this.comments = new issues.CommentsCollection({page: 1});`